### ISSUE TITLE
Esteban Mendoza - Lightning Presentation

### ISSUE THREAD

**Issue by estmendoza** [ShawnTylerSchwartz/eeb-c177-w20#77]
https://drive.google.com/open?id=1WgA9zCTQ9sZDLjxcw1hk4LyFuuNyHmND

**Comment by estmendoza** [comment]
or this https://youtu.be/IqTPqYO1S88

**Comment by LinhN16** [comment]
This is an incredibly practical dataset to analyze and the results of this could be extremely beneficial for future health care policies! I wonder the efficiency of datasets like these and how much weight they hold in policy making. I also enjoyed that your live code demonstration was able to produce bar graphs that make it easier for the user to visualize data.

**Comment by haonguyen318** [comment]
I liked how you highlighted the challenges that you faced when graphing your datasets. I also liked that you because the first graphs that you showed contained a lot of data, it was hard to follow along and see each components of the graph, you made a more zoomed in versions of those graphs and it was much easier to see. I also enjoyed watching your lab demonstration as you worked out how you created your graph with the codes that you used. You also created a code to exclude things that are non DSM which I thought was a great idea. Overall, it was a very good presentation!

**Comment by alexphu1230** [comment]
Great job Esteban! I like how you analyzed the results and its practical implications. I also thought that your visualizations for the data was very helpful in understanding your dataset and the manipulations that you did to understand your dataset. Overall I thought it was a super helpful and interesting presentation!

**Comment by goharmihranian** [comment]
I thought it was so cool that your program runs from the shell and produces bar graphs upon categorical inputs! This is a super useful dataset because it can really help in public funding. I thought it was useful that you included the exclude option because, like you said, no one is going to want to pay for someone else's cosmetic procedures; whereas something like depression is serious and people actually need treatment for that.

**Comment by jessicadeanda** [comment]
Great analysis Esteban! I really like that you questioned the factors influencing the statistical significance of your initial analysis and that you addressed this by excluding cosmetic procedures. Since the prevalence of a condition may affect whether or not the condition is deemed "deserving of funding", it may be beneficial to compare the prevalence of these conditions in Finland versus the US.

**Comment by altaing** [comment]
Hi Esteban,
I agree with Gohar that it was really cool that you got your program to run in the shell! I thought your data set was really interesting in that it really highlights the differences in how people think depending on what information they have. For your graphs, did you make them using only python or did you use R/ggplot?

**Comment by jpocon** [comment]
Hi Esteban! I have to agree with everyone here, the interactive nature of your program is really well executed and covering live demo(s) in under three minutes was impressive. Great job. My (two-part) question is about the output and how you see yourself applying your code after the course: 1) Does the program save the plots as well? and 2) is the shell + script method what you plan to use on your personal OS? Thanks!

**Comment by fridalejandra** [comment]
Esteban, what an important data set to analyze. I thought this was a great presentation, the interactive shell was great ! What was your main usage in python, was it cleaning the data?

**Comment by estmendoza** [comment]
@jessicadeanda definitely! I also think it would be super cool to compare perceptions between Finland and the US, but I don't think data like this exists for the US yet :/ 

@altaing used R/ggplot for first graph (the prettier one with all the states of being) and python for the demo graphs (the green bar plots that aren't as pretty) (: 
This is my code for the graph made in R: https://github.com/estmendoza/eeb-c177-projects/blob/master/analyses/average-chart.R

@jpocon 1) my code doesn't save the plots, but I think including that would be useful! 
2) I figured I'd run it in the shell since we learned how to do that early on in class, so I just ran with that 😅

@fridalejandra yes! a lot of my time was spent trying to set up the data in a way I felt comfortable using. and creating if/else statements based on user input. I looove asking for input. 
Forgot to include it, but here's my Python code: https://github.com/estmendoza/eeb-c177-projects/blob/master/analyses/modifying%20project%20script.ipynb

**Comment by dylanreadel** [comment]
Loved the extremely detailed and thought out analysis. I have practically no education in anything medical related, so some of this was over my head, but I did understand the importance of it. Just from one of your plots, I was able to tell that doctors and nurses tended to rank most states of being as diseases and needing public funding more so than the layperson/parliament. Much of the public/political world does not understand the severity of many diseases which can prevent funding from being allocated, so I appreciate the work you've done.

**Comment by soniavsd** [comment]
Hey Esteban! I really liked how you used an interactive shell! Very impressive:) I think it would be very interesting to see what would happen if the survey was operated across smaller areas to see what types of people feel about specific illnesses like depression
